products/views.py

```diff
@@ -83,18 +83,40 @@ class ProductReview(View):
     @authorization
     def post(self, request):
         data = json.loads(request.body)
-
         try:
             if not Product.objects.filter(id=data['product_id']).exists():
                 return JsonResponse({'MESSAGE':'INVALID_ERROR'},status=401)
-
             Review.objects.create(
                 user       = request.user,
                 product_id = data['product_id'],
                 comment    = data['comment'],
                 score      = data['score']
             )
-            
             return JsonResponse({'MESSAGE':'SUCCESS'}, status=201)
         except KeyError:
-            return JsonResponse({'MESSAGE':'KEYERROR'}, status=400)
\ No newline at end of file
+            return JsonResponse({'MESSAGE':'KEYERROR'}, status=400)
+
+class SearchView(View):
+    def post(self, request):
+        try:
+            page        = int(request.GET.get('page', 1))
+            page_size   = 24
```

Comment:
상수화 시킨 변수는 보통 대문자로 작성합니다!
```suggestion
            PAGE_SIZE   = 24
```

Reply:
변경했습니다! 구매하기는 성준님께서 구현하시지만 함께 생각해보도록 하겠숩니다! ㅎㅎ